### PR TITLE
Fix Bug 1018373 - Add tooltip to Abandon and Complete task

### PR DESCRIPTION
**Pull request by tessie**

### CODE REVIEW

**Comment by bobsilverberg** [comment]
@tessie Tooltips look good, but we need to localize the strings. You can use the same syntax as is used for the button text, e.g., 

`title="{{ _('Abandon this task') }}"`

**Comment by tessie** [comment]
@bobsilverberg .fixed

**Comment by bobsilverberg** [comment]
Nice work, @tessie. :)

Please squash the commits and I will merge it.

**Comment by tessie** [comment]
@bobsilverberg .squashed

**Comment by bobsilverberg** [comment]
Great, thanks @tessie!

File: oneanddone/tasks/templates/tasks/detail.html [hunk 1]
```diff
@@ -123,12 +123,12 @@
 
       <form method="post" action="{{ url('tasks.abandon', attempt.id) }}">
         {{ csrf() }}
-        <button type="submit" class="button" id="abandon-task">{{ _('Abandon task') }}</button>
+        <button type="submit" title="{{ _('Abandon this task') }}" class="button" id="abandon-task">{{ _('Abandon this task') }}</button>
```
Comment (bobsilverberg) [inline oneanddone/tasks/templates/tasks/detail.html]: @tessie The word `this` was only meant to be added to the tooltip, not to the button text, so this is incorrect. Can you please fix it and issue a new PR?

The same goes for line 131 below.

**Pull request merged**